src/plugins/intel_gpu/src/graph/graph_optimizer/prepare_primitive_fusing.cpp

```diff
@@ -992,7 +988,8 @@ void prepare_primitive_fusing::fuse_simple_primitives(program &p) {
             auto p2_dt = parents[peer_idx].first->get_output_layout().data_type;
 
             if (can_fuse_parents[peer_idx] &&
-               ((p1_pnum < p2_pnum && p1_dt == p2_dt) || (data_type_traits::is_floating_point(p2_dt) && !data_type_traits::is_floating_point(p1_dt)))) {
+               ((p1_pnum < p2_pnum && fused_idx > peer_idx && p1_dt == p2_dt) ||
```

Comment:
I think this code change is so limited workaround code. if the order fused_node_idx and peer_node_idx are swapped, same issue will be occurred in this condition.

Reply:
Changed to check if the parent is in fusing history